### PR TITLE
Added more custom coloring of layers

### PR DESCRIPTION
**Pull request by frederique-hub**

### CODE REVIEW

**Review by Santonia27 — APPROVED** [review]

File: src/guitares/pyqt5/tableview.py
Comment (panosatha) [inline src/guitares/pyqt5/tableview.py:1]: this seems to be creating some inconsistensies on how the tables are created in the FloodAdaptGUI. We need to check on how to make this flexible and not hard-coded for one specific GUI.

Comment (panosatha) [reply src/guitares/pyqt5/tableview.py:1]: corrected that now

**Comment by panosatha** [comment]
The issue with the table widths is now resolved. I added a threshold of 1000 elements in a table to use one approach over the other.

**Comment by panosatha** [comment]
I will merge this pull request so we have a single branch that works for both FloodAdapt and the ModelBuilder, **BUT** I do believe we need to refactor the way that some of these layers have been created, since basically now Frederique made 3 new layers that are essentially similar to the ones that were already there but with custom options. The correct way would be to just have a single layer per type and offer all these options as optional arguments.

E.g., there should be only one circle layer and one choropleth (polygon) layer.

I will create new issues for these matters.

@Santonia27 can you please check if this updates works fine for the Model Builder GUI? I cannot test since my environemnt is not running.

**Comment by Santonia27** [comment]
I agree with Panos to refactor to single layers per type. 
The model builder works fine and  NSI data tables (> 1000 rows) load fast. So, ready to merge!

**Pull request merged**